common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -66,7 +66,7 @@ data: |
               width="400"
               height="400"
               html_file="https://studio.edx.org/c4x/edX/DemoX/asset/webGLDemo.html"
-              title="Cone and Cube Frame"
+              title="Spinning Cone and Cube"
```

Comment:
I updated the file stored at https://studio.edx.org/c4x/edX/DemoX/asset/webGLDemo.html to have a title element, matching this value.